src/userobjects/MoabSkinner.C

```diff
@@ -273,6 +294,10 @@ MoabSkinner::update()
   // Clear MOAB mesh data from last timestep
   reset();
 
+  if (_use_displaced && _fe_problem.getDisplacedProblem() != nullptr)
```

Comment:
Should we throw an error if the user writes `use_displaced = true` but `getDisplacedProblem` returns a null ptr? I think that could go in the constructor, and then we'd not have to check this nullptr here.